File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--programmatically-search-for-and-replace-text--in-documents_9.cs
        private void RangeFind() 
        { 
            object findText = "find me";

            Word.Range rng = this.Paragraphs[2].Range; 

            rng.Find.ClearFormatting();

            if (rng.Find.Execute(ref findText,
                ref missing, ref missing, ref missing, ref missing, ref missing, ref missing,
                ref missing, ref missing, ref missing, ref missing, ref missing, ref missing, 
                ref missing, ref missing)) 
            { 
                MessageBox.Show("Text found.");
            } 
            else 
            { 
                MessageBox.Show("Text not found.");
            } 

            rng.Select(); 
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--programmatically-search-for-and-replace-text--in-documents_9.vb
    Private Sub RangeFind()
        Dim findText As String = "find me"

        Dim rng As Word.Range = Me.Paragraphs(2).Range

        rng.Find.ClearFormatting()

        If rng.Find.Execute(findText) Then
            MessageBox.Show("Text found.")
        Else
            MessageBox.Show("Text not found.")
        End If

        rng.Select()
    End Sub